src/current/cockroachcloud/cluster-management.md

```diff
@@ -129,13 +126,14 @@ You can add up to nine regions at a time through the Console. See [Planning your
 
     The **Edit cluster** page displays.
 
-1. Click **Add a region**.
+1. On the **Regions** page, click **Add a region**.
 
     If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions.
 
-1. In the **Regions & Nodes** section, select the desired new region and specify the number of nodes for it.
-1. In the **Summary** sidebar, verify the hourly estimated cost for the cluster.
-1. Click **Update**.
+1. Select the desired new region and specify the number of nodes for it.
+1. In the sidebar, verify the hourly estimated cost for the cluster.
+1. Click **Next: Capacity** and then click **Finalize**.
```

Comment:
Same question about whether you need to select anything in Capacity to get to the Finalize step.

Reply:
Nope!